packages/env/lib/build-docker-compose-config.js

```diff
@@ -240,9 +235,19 @@ module.exports = function buildDockerComposeConfig( config ) {
 			},
 			'tests-wordpress': {
 				depends_on: [ 'tests-mysql' ],
-				image: testsWpImage,
+				build: {
+					context: '.',
+					dockerfile: 'Tests-WordPress.Dockerfile',
+					args: {
```

Comment:
Possibly worth deduplicating some of the config, since we have to define some of this stuff 4 times